packages/language-server/src/documentation.ts

```diff
@@ -39,8 +39,24 @@ export function getNodeDocumentation(
       : EMPTY_STRING;
     contents += NL;
   }
-  // TODO add experimental when it's added to the model in this format: Experimental API [since version <since>]
 
+  if (node.experimentalInfo?.isExperimental) {
+    /* istanbul ignore else */
+    if (contents !== "") {
+      contents += NL;
+    }
+    contents += "Experimental";
+    /* istanbul ignore next */
+    contents += node.experimentalInfo.since
```

Comment:
Use Nullish coalescing instead of the ternary operator.

Reply:
It should go to the `EMPTY_STRING` case also when `since` is an empty string

Reply:
- The result would be the same.
- The verbosity would be reduced.
- The semantics of the branch flow can be defined in multiple ways the specific branch
  chosen when X is an empty string is less relevant as long as the result is correct.

```javascript
const EMPTY_STRING = "";
const X = "";
// Print X unless X is undefined / Null
// otherwise Print EMPTY_STRING
console.log(X ?? EMPTY_STRING) // ""

// Print X unless X is undefined / Null / Empty / Empty Array / 0 / and the PRINT
// otherwise Print EMPTY_STRING
console.log(X ? X : EMPTY_STRING) // ""
```

Reply:
Right, but we're not concatenating X here but a string that contains X

Reply:
If we don't check the case where `since` is an empty string we'll concatenate "` since version `" (without a version number) to the content.

Reply:
ah I see, so lets move this comment to the concatnating of the "text" field below :)

Reply:
It's less visible but there's an extra space there that will be added in that case

Reply:
o.k. I see the `??` is not suitable here 😄